geode-core/src/distributedTest/java/org/apache/geode/internal/cache/execute/PRClientServerTestBase.java

```diff
@@ -273,12 +259,12 @@ public static void createCacheClient(String host, Integer port1, Integer port2,
     factory.setScope(Scope.LOCAL);
     factory.setDataPolicy(DataPolicy.EMPTY);
     factory.setPoolName(p.getName());
-    RegionAttributes attrs = factory.create();
-    Region region = cache.createRegion(PartitionedRegionName, attrs);
+    RegionAttributes<?, ?> attrs = factory.create();
```

Comment:
We could use Object, Object instead of ?,? because we only Objects(String, Integer, Boolean) in region.